http_2.rflx

```diff
@@ -0,0 +1,238 @@
+package HTTP_2 is
+
+   type Payload_Length is mod 2**24;
+
+   type Frame_Type is (
+      DATA => 0,
+      HEADERS => 1,
+      PRIORITY => 2,
+      RST_STREAM => 3,
+      SETTINGS => 4,
```

Comment:
> > Rebased onto main, but there seems to be a problem in the tests with the newly added `Settings` field.
> 
> It seems there is a name conflict in the generated code. Such issues should be detected during model verification. Please create an issue for RecordFlux, preferably with a small reproducer.

Just to make sure that this does not get lost.

Reply:
https://github.com/Componolit/RecordFlux/issues/520